CMakeLists.txt

```diff
@@ -14,25 +14,32 @@ endif()
 include (CTest)
 enable_testing()
```

Comment:
Off-topic: should this be enabled only when tests or examples are enabled?

Reply:
Fixed